src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionModeller.java

```diff
@@ -42,30 +42,43 @@
     private final List<Double> outlierProbabilitySamples = new ArrayList<>();
     private final List<AlleleFractionState.MinorFractions> minorFractionsSamples = new ArrayList<>();
     private final int numSegments;
 
+    public AlleleFractionModeller(final SegmentedGenome segmentedGenome) {
+        this(segmentedGenome, AllelicPanelOfNormals.EMPTY_PON);
+    }
+
     public AlleleFractionModeller(final SegmentedGenome segmentedGenome, final AllelicPanelOfNormals allelicPON) {
         this.segmentedGenome = segmentedGenome;
+        this.allelicPON = allelicPON;
         final AlleleFractionData data = new AlleleFractionData(segmentedGenome, allelicPON);
         numSegments = data.getNumSegments();
         final AlleleFractionState initialState = new AlleleFractionInitializer(data).getInitializedState();
 
         // Initialization got us to the mode of the likelihood
         // if we approximate conditionals as normal we can guess the width from the curvature at the mode and use as the step size
+        final AllelicBiasParameters initialParameters = initialState.getParameters();
+        final AlleleFractionState.MinorFractions initialMinorFractions = initialState.minorFractions();
 
         final double meanBiasInitialStepSize = getStepSizeFromApproximatePosteriorWidthAtMode(meanBias ->
-                AlleleFractionLikelihoods.logLikelihood(initialState.shallowCopyWithProposedMeanBias(meanBias), data), initialState.meanBias());
+                AlleleFractionLikelihoods.logLikelihood(initialParameters.copyWithNewMeanBias(meanBias), initialMinorFractions, data), initialParameters.getMeanBias());
         final double biasVarianceInitialStepSize = getStepSizeFromApproximatePosteriorWidthAtMode(biasVariance ->
-                AlleleFractionLikelihoods.logLikelihood(initialState.shallowCopyWithProposedBiasVariance(biasVariance), data), initialState.biasVariance());
+                AlleleFractionLikelihoods.logLikelihood(initialParameters.copyWithNewBiasVariance(biasVariance), initialMinorFractions, data), initialParameters.getBiasVariance());
         final double outlierProbabilityInitialStepSize = getStepSizeFromApproximatePosteriorWidthAtMode(outlierProbability ->
-                AlleleFractionLikelihoods.logLikelihood(initialState.shallowCopyWithProposedMeanBias(outlierProbability), data), initialState.outlierProbability());
-        final List<Double> minorFractionsInitialStepSizes = IntStream.range(0, numSegments).mapToDouble(segment ->
-                getStepSizeFromApproximatePosteriorWidthAtMode(AlleleFractionLikelihoods.segmentLogLikelihoodConditionalOnMinorFraction(initialState, data, segment), initialState.segmentMinorFraction(segment)))
+                AlleleFractionLikelihoods.logLikelihood(initialParameters.copyWithNewOutlierProbability(outlierProbability), initialMinorFractions, data), initialParameters.getOutlierProbability());
+        
+
+/* Sam
```

Comment:
Whoops, left that in as a note to self from a tricky part of the rebase where we had modified the same line.  I wanted to be sure I didn't revert your PR by accident.